src/merge.js

```diff
@@ -19,6 +19,9 @@ var _extend = require('./internal/_extend');
  *      R.merge({ 'name': 'fred', 'age': 10 }, { 'age': 40 });
  *      //=> { 'name': 'fred', 'age': 40 }
  *
+ *      R.merge({ 'name': 'fred', 'age': 10 }, { 'age': undefined });
+ *      //=> { 'name': 'fred', 'age': undefined }
```

Comment:
> I'm curious as to why one might expect `undefined` to be treated differently from other values. Perhaps some other libraries have set a bad precedent. ;)

That, and JS itself is extremely schizophrenic about `undefined`.  It's the value that says, "there's no value, damn it, never has been and perhaps never will be."  Its cousin, `null`, is the more sane one that simply says, "this space intentionally left blank".

But I agree with the overall point that there's no compelling reason to document that `{age: undefined}` will override an `age` field with another value.  Why would one expect otherwise?